ui/components/src/Tooltip/SeriesInfo.tsx

```diff
@@ -109,20 +117,21 @@ export function SeriesInfo(props: SeriesInfoProps) {
           maxWidth: '520px',
         }}
       >
-        <SeriesMarker markerColor={markerColor} />
+        <SeriesMarker markerColor={markerColor} sx={{ marginTop: 0.5 }} />
         <Box
           component="span"
           sx={(theme) => ({
             color: theme.palette.common.white,
             display: 'inline-block',
+            width: 'calc(100% - 20px)',
+            minWidth: 150, // TODO: use clamp instead
             maxWidth: TOOLTIP_LABELS_MAX_WIDTH,
             overflow: 'hidden',
             textOverflow: 'ellipsis',
             whiteSpace: wrapLabels ? 'normal' : 'nowrap',
-            width: 'calc(100% - 20px)',
           })}
         >
-          {inlineSeriesLabels}
+          <Typography variant="body2">{inlineSeriesLabels}</Typography>
```

Comment:
Noticed that fonts aren't quite right so I'm planning to fix this (I need to figure out what variant to use for the formattedY value), but the rest of this PR is ready for review.